payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardDetailsSectionElementUI.kt

```diff
@@ -0,0 +1,117 @@
+import com.stripe.android.ui.core.DefaultIsStripeCardScanAvailable
+import com.stripe.android.ui.core.PaymentsTheme
+import com.stripe.android.ui.core.R
+
+@Composable
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun CardDetailsSectionElementUI(
+    enabled: Boolean,
+    hiddenIdentifiers: List<IdentifierSpec>?,
+    context: Context
+) {
+    val cardDetailsElement = CardDetailsElement(
+        IdentifierSpec.Generic("credit_detail"), context
+    )
+
+    val isStripeCardScanAvailable = DefaultIsStripeCardScanAvailable()
+
+    Row(
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(top = 9.dp),
+    ) {
+        H6Text(
+            text = stringResource(R.string.card_information),
+            modifier = Modifier
+                .semantics(mergeDescendants = true) { // Need to prevent form as focusable accessibility
+                    heading()
+                }
+        )
+        if (isStripeCardScanAvailable()) {
+            val cardScanLauncher =
+                rememberLauncherForActivityResult(ActivityResultContracts.StartActivityForResult()) {
+                    if (it.data != null) {
+                        cardDetailsElement.controller.numberElement.controller.onCardScanResult(
+                            it.data!!.getParcelableExtra(CARD_SCAN_PARCELABLE_NAME)
+                                ?: CardScanSheetResult.Failed(
+                                    UnknownScanException("No data in the result intent")
+                                )
+                        )
+                    }
+                }
+            Row(
+                verticalAlignment = Alignment.CenterVertically,
+                modifier = Modifier.clickable(
+                    indication = null,
+                    interactionSource = remember { MutableInteractionSource() },
+                    onClick = {
+                        cardScanLauncher.launch(
+                            Intent(
+                                context,
+                                CardScanActivity::class.java
+                            )
+                        )
+                    }
+                )
+            ) {
+                Image(
+                    painter = painterResource(R.drawable.stripe_ic_camera),
+                    contentDescription = stringResource(
+                        R.string.scan_card
+                    ),
+                    colorFilter = ColorFilter.tint(PaymentsTheme.colors.material.primary)
+                )
+                Text(
+                    stringResource(R.string.scan_card),
+                    Modifier
+                        .padding(start = 4.dp),
+                    color = PaymentsTheme.colors.material.primary,
+                    style = PaymentsTheme.typography.h6,
+                    fontWeight = FontWeight.Bold
```

Comment:
style will set the font weight. We don't need to bold it here.

Reply:
H6 typography isn't bolded though